package.json

```diff
@@ -1,54 +1,29 @@
 {
   "name": "hubot",
   "version": "2.19.0",
-  "author": "hubot",
   "keywords": [
     "github",
     "hubot",
     "campfire",
     "bot"
   ],
   "description": "A simple helpful robot for your Company",
-  "license": "MIT",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/github/hubot.git"
   },
   "dependencies": {
     "hubot": "^2.19.0",
-    "async": ">=0.1.0 <1.0.0",
-    "chalk": "^1.0.0",
-    "cline": "^0.8.2",
-    "coffee-script": "1.6.3",
-    "connect-multiparty": "^1.2.5",
-    "express": "^3.21.2",
-    "log": "1.4.0",
-    "optparse": "1.0.4",
-    "hubot-front": "^0.0",
-    "scoped-http-client": "0.11.0"
+    "hubot-duplicate": "github:resin-io-modules/hubot-duplicate#alpha",
+    "hubot-flowdock": "github:resin-io-modules/hubot-flowdock#alpha"
   },
   "devDependencies": {
     "resin-lint": "^1.4.0",
-    "hubot-mock-adapter": "^1.0.0",
-    "hubot-flowdock": "",
-    "hubot-slack": "",
-    "coffee-errors": "0.8.6",
-    "mocha": "^2.1.0",
-    "mockery": "^1.4.0",
-    "sinon-chai": "^2.8.0",
-    "sinon": "~1.17.0",
-    "chai": "~2.1.0"
+    "hubot-mock-adapter": "^1.0.0"
   },
   "engines": {
-    "node": ">= 0.8.x",
-    "npm": ">= 1.1.x"
-  },
-  "main": "./index",
-  "bin": {
-    "hubot": "./bin/hubot"
-  },
-  "scripts": {
-    "start": "./bin/hubot -a front -n webot",
-    "test": "./script/test"
+    "node": "^6.9.4",
+    "npm": "^4.1.1"
```

Comment:
It'd be nice to keep bot startup managed by npm. Means there's a single main place to go to trigger actions (`npm install/start/test`), it unties execution from heroku, and it's more consistent with all the rest of our projects. If you move the script in the Procfile into here, and just run `npm start` in the Procfile, everything should work nicely.